gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -133,6 +133,13 @@
               </div>
             </div>
           </div>
+          <div class="row">
+            <div class="form-group col-md-6">
+              <input type="checkbox" id="untrusted" ng-model="currentDevice.untrusted" />
+              <label for="untrusted" translate>Untrusted</label>
+              <p translate class="help-block">All folders shared with this device must be protected by a password, such that all sent data is encrypted.</p>
```

Comment:
`such that all sent data is unreadable without the given password`?

Encrypted might just mean TLS...